hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/keyvalue/impl/AbstractTestChunkManager.java

```diff
@@ -128,6 +135,47 @@ protected void checkChunkFileCount(int expected) {
     assertEquals(expected, files.length);
   }
 
+  /**
+   * Helper class to check if a file is in use.
+   */
+  public static class FuserCheck {
```

Comment:
Hi Wei, whats the reason behind using this as a class name?
FileUsageChecker or FileInUseChecker sounds more apt to me

Reply:
removed the static class. Just use the method itself.